### PR TITLE
Hlin explosive compensation module

### PR DESCRIPTION
**Pull request by TiviPlus**
## About The Pull Request

Does what you'd expect

## Why It's Good For The Game

Mhmm customization

## Changelog
:cl:
add: Hlin explosive compensation module
/:cl:

### CODE REVIEW

**Comment by jroinc** [comment]
100% explosion immunity is an OOF cause it opens up retarted strategies as a manner of course.

As funny as EXPLOSION MAN, the guy with 27 frag grenades who can be contained by no hive could be.

or PB SADAR man.

Drop it to 90 or 95% overall? Still obscenely good, but you do take a coupla points of damage.

File: code/modules/reqs/supplypacks.dm [hunk 1]
```diff
@@ -833,6 +834,12 @@ ARMOR
 	)
 	cost = 12
 
+/datum/supply_packs/armor/modular/attachments/mimir_environment_protection
```
Comment (InterroLouis) [inline code/modules/reqs/supplypacks.dm:837]: Needs new pathing for the orderable.

**Comment by TiviPlus** [comment]
Do note this does not protect the head or provide flash immunity

**Comment by RenGusta** [comment]
Should call it Hlin cause Freyja wasn't about protection or whatever. Hlin is a norse goddess of protection.

**Comment by SilencedMP5A5** [comment]
where, WHeRE iS IT

**Review by Hughgent — APPROVED** [review]

**Pull request merged**